packages/blade/src/components/Table/__tests__/__snapshots__/Table.web.test.tsx.snap

```diff
@@ -1157,6 +1165,8 @@ exports[`<Table /> should render table 1`] = `
           </td>
         </tr>
         <tr
+          aria-multiselectable="false"
```

Comment:
aria-multiselectable will go on the parent and aria-selected will be there on child right 🤔 ?

Reply:
Yeah, this should go on table instead

Reply:
Updated